app/views/lessons/_header.html.erb

```diff
@@ -3,9 +3,16 @@
     <%= render Course::BadgeComponent.new(course: lesson.course, options: {show_badge: true }) %>
 
     <div class="text-center xl:text-left">
-      <h1 class="text-2xl font-semibold text-gray-800 dark:text-gray-200 min-w-full" data-test-id="lesson-title-header">
-        <%= lesson.display_title %>
-      </h1>
+      <div class="flex items-center gap-3">
+        <h1 class="text-2xl font-semibold text-gray-800 dark:text-gray-200" data-test-id="lesson-title-header">
+          <%= lesson.display_title %>
+        </h1>
+        <% if lesson.recently_added? %>
```

Comment:
Nit: Seeing a little bit of weirdness on smaller screens. We could maybe break the badge to a new line on the sm breakpoint and below

<img width="373" alt="Screenshot 2024-06-11 at 23 54 02" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/c73598c1-12ac-448b-aca9-99639f1b2ba0">

Reply:
nice catch!

Reply:
Thanks @01zulfi - looks perfect